features/generate.feature

```diff
@@ -10,6 +10,15 @@ Feature: Generate a new project using Tuist
     Then I should be able to test the scheme AppUITests
     Then I should be able to build the scheme App-Manifest
 
+  Scenario: The project is an iOS application with custom storyboard names (ios_app_with_custom_storyboard_names)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture ios_app_with_custom_storyboard_names into the working directory
+    Then tuist generates the project
+    Then I should be able to build the scheme App
+    Then the product 'App.app' with destination 'Debug-iphoneos' has an entry of 'UIMainStoryboardFile' in the info.plist with the value of 'Custom Main'
```

Comment:
Given that this type of tests are harder to maintain and take longer to execute, I'd merge this one into an existing test.